src/panes.ts

```diff
@@ -55,7 +55,7 @@ export type RedirectPane = Pane<
   {
     redirect_url: string
     pre_redirect_acknowledgement?: boolean
-    context?: "smartthings_auth"
+    context?: "smartthings_auth" | "oauth_redirect"
```

Comment:
🐧 personally I like something like `does_redirect_disallow_embeds: boolean` instead of extending the enum

Reply:
ok changed to flag `redirect_target_disallows_embedding`